components/mediaUploader.css

```diff
@@ -264,6 +264,8 @@ md-dialog {
   text-align: center;
   opacity: 1;
 }
+.md-block.md-icon-float.md-input-has-placeholder > label{
```

Comment:
This needs to be a little more specific for your use case, or this `display:none` will bleed into other components.